tests/test-sources/plugins/by-name/otter/default.nix

```diff
@@ -15,7 +25,7 @@
       # Avoid the warning
       treesitter = {
         enable = true;
-        settings.highlight.enable = true;
```

Comment:
Why is this removed if it's an "old api" test case

Reply:
This is the defaults test case